### PR TITLE
feat: added log aggregation in grafana for local services

### PR DESCRIPTION
**Pull request by johnrwatson**
Four main changes:
- Adds a `/tmp/si-logs/<service>` log file when running tilt. Deletes/Recreates this folder just before starting tilt. 
- `tees` the logs of the rust services into those files
- Adds promtail + loki so that the service logs for the rust services can be aggregated in local grafana.
- Sorts out the groupings of the services within the tiltfile

Kudos should be directed to my partner in crime @keeb

### CODE REVIEW

File: dev/docker-compose.platform.yml [hunk 1]
```diff
@@ -118,7 +158,7 @@ services:
       - jaeger
 
   localstack:
+    #<<: *loki-logged-service
     image: localstack/localstack
     ports:
       - "4566:4566"
-      - "4510-4559:4510-4559"
```
Comment (johnrwatson) [inline dev/docker-compose.platform.yml:124]: removed as these are not needed for local development and were just hogging the tilt UI space

File: prelude-si/tilt.bzl [hunk 2]
```diff
@@ -124,12 +124,16 @@ def _invoke_tilt(ctx: AnalysisContext, subcmd: str) -> list[[DefaultInfo, RunInf
         ctx.attrs.tiltfile,
     )
 
+    RunInfo(["rm", "-rf", "/tmp/si-logs"])
```
Comment (johnrwatson) [inline prelude-si/tilt.bzl:127]: Nothing like an `rm -rf` to wake you up in the morning

Comment (keeb) [reply prelude-si/tilt.bzl:127]: This one is tactical

**Comment by johnrwatson** [comment]
bors merge

**Pull request merged**